samples/bluetooth/mesh/light_ctrl/src/model_handler.c

```diff
@@ -242,11 +260,21 @@ static struct bt_mesh_elem elements[] = {
 	BT_MESH_ELEM(1,
 		     BT_MESH_MODEL_LIST(
 			     BT_MESH_MODEL_CFG_SRV,
-			     BT_MESH_MODEL_HEALTH_SRV(&health_srv, &health_pub),
+#if defined(CONFIG_BT_MESH_LARGE_COMP_DATA_SRV)
```

Comment:
I do not think condition compilation is required for any models except `BT_MESH_MODEL_LARGE_COMP_DATA_SRV`. If there is no Large composition data server, then metadata will go nowhere. There shouldn't be any restrictions to do so.

Reply:
Will give it a try :)